service/src/main/java/aero/minova/cas/SecurityConfig.java

```diff
@@ -53,48 +55,52 @@ public class SecurityConfig {
 	@Value("${server.port:8084}")
 	private String serverPort;
 
-	private final Environment environment;
-	private final CustomLogger customLogger;
-	private final DataSource dataSource;
+	@Value("${cors.allowed.origins:http://localhost:8100,https://localhost:8100}")
```

Comment:
Einfach im Hinterkopf behalten: In einer `application.yml` bzw. `application.properties` könnten eigene Konfigurationen in der Hierarchie: "application:" zusammengefasst werden. Das separiert die eigene Konfiguration von dem SpringBoot o.ä. Gedöns. Nicht, dass das tatsächlich auch bei CAS durchgeführt wurde, aber schöner wäre das.